src/main/java/sim/views/SimulationGUI.java

```diff
@@ -97,7 +99,8 @@ private JPanel setupControlls() {
 
 	// Observes when shop is finished simulation
 	public void update() {
-		// TODO
+		JOptionPane.showMessageDialog(this, "The simulation has now ended. \nEnd of day report and log saved to file.");
```

Comment:
Only thing I'd maybe do is include the file names in the message string for convenience

Reply:
Did think of that but thought the message was getting a bit long but fair point about the convenience so I'll just add that in.